examples/client/Locomotion/android/app/proguard-rules.pro

```diff
@@ -8,3 +8,6 @@
 #   http://developer.android.com/guide/developing/tools/proguard.html
 
 # Add any project specific keep options here:
+
+-keep class com.locomotion.BuildConfig { *; }
+-keep class com.google.android.gms.location.** { *; }
```

Comment:
what it affects?

Reply:
it was used to make the build work locally but now it works without it using patches